src/AWS/Orleans.Streaming.SQS/Hosting/ClientBuilderExtensions.cs

```diff
@@ -14,32 +14,36 @@ public static class ClientBuilderExtensions
         /// </summary>
         public static IClientBuilder AddSqsStreams(this IClientBuilder builder, string name, Action<SqsStreamOptions> configureOptions)
         {
-            return builder.ConfigureServices(services => services.AddClusterClientSqsStreams(name, configureOptions));
+            return builder
+                .ConfigureApplicationParts(parts => parts.AddFrameworkPart(typeof(SQSAdapterFactory).Assembly))
+                .ConfigureServices(services => services.AddClusterClientSqsStreams(name, configureOptions));
         }
 
         /// <summary>
         /// Configure cluster client to use SQS persistent streams.
         /// </summary>
         public static IClientBuilder AddSqsStreams(this IClientBuilder builder, string name, Action<OptionsBuilder<SqsStreamOptions>> configureOptions = null)
         {
-            return builder.ConfigureServices(services => services.AddClusterClientSqsStreams(name, configureOptions));
+            return builder
+                .ConfigureApplicationParts(parts => parts.AddFrameworkPart(typeof(SQSAdapterFactory).Assembly))
+                .ConfigureServices(services => services.AddClusterClientSqsStreams(name, configureOptions));
         }
 
         /// <summary>
         /// Configure cluster client to use SQS persistent streams.
         /// </summary>
-        public static IServiceCollection AddClusterClientSqsStreams(this IServiceCollection services, string name, Action<SqsStreamOptions> configureOptions)
+        private static void AddClusterClientSqsStreams(this IServiceCollection services, string name, Action<SqsStreamOptions> configureOptions)
```

Comment:
why are we making those methods private?

Reply:
From the linked issue: https://github.com/dotnet/orleans/issues/4084#issuecomment-369024506

In other discussions (I started an email thread on it) I mentioned that IServiceCollection is an ambiguous place for most silo-specific or client-specific extension methods

Reply:
I see

Reply:
Making these private is sufficient, but my preference would be to delete them entirely and do the work in one of the builder calls.  At this point, these service collection overrides are just excess cruft now, imo.